src/Views/batchprocessor.html

```diff
@@ -275,22 +275,38 @@ <h3>Select Basin Characteristics</h3>
 
 <!-- Start modal footer -->
 <div class="modal-footer">
-    <div class="row" style="justify-content: center;">
+    <div ng-if="vm.selectedBatchProcessorTabName == 'submitBatch' && vm.submitBatchOver250" class="bp-status-250" >
+        {{vm.submitBatchOver250Message}}
+    </div>
+    <div class="row" style="justify-content: center; padding-top: 15px;">
         <!-- Button to submit batch -->
-        <button ng-if="vm.selectedBatchProcessorTabName==='submitBatch'" type="submit" class="btn-wim-default" style="margin-right: 10px;" ng-click="vm.submitBatch()"
-            ng-disabled="vm.submittingBatch || !vm.selectedRegion || !vm.submitBatchData.idField || !vm.submitBatchData.email || !vm.submitBatchData.attachment"><i
-
-                class="fa fa-share"></i><span>&nbsp;&nbsp;Submit Batch</span></button>
+        <button ng-if="vm.selectedBatchProcessorTabName==='submitBatch' && vm.submitBatchOver250 == false" type="submit" class="btn-wim-default" style="margin-right: 10px;" ng-click="vm.submitBatch()"
+            ng-disabled="vm.submittingBatch || !vm.selectedRegion || !vm.submitBatchData.idField || !vm.submitBatchData.email || !vm.submitBatchData.attachment">
+            <i ng-if="vm.submittingBatch == false" class="fa fa-share"></i>
+            <i ng-if="vm.submittingBatch == true" class="fa fa-spinner fa-3x fa-spin loadingSpinner"></i>

```

Comment:
Can we use this spinner instead? This is what we use for other buttons in the app
```suggestion
            <i ng-if="vm.submittingBatch == true" class="fa fa-refresh fa-spin"></i>

```